common/get-host-info.sub.js

```diff
@@ -0,0 +1,19 @@
+function get_host_info() {
+
+  var HTTP_PORT = eval('{{ports[http][0]}}');
+  var HTTP_PORT2 = eval('{{ports[http][1]}}');
+  var HTTPS_PORT = eval('{{ports[https][0]}}');
+  var ORIGINAL_HOST = eval('\'{{host}}\'');
```

Comment:
Right, that is a left-over from the initial file that comes from Chromium (see /fetch/api/resources/get-host-info.sub.js).